cli/src/commands/schema/edit.ts

```diff
@@ -734,24 +762,49 @@ function validateOptionalBoolean(value?: string) {
   return true;
 }
 
+function validateUnique(uniqueValue: string | undefined, state: ColumnEditState) {
+  const isUnique = parseBoolean(uniqueValue);
+  if (isUnique && state.values.type && uniqueUnsupportedTypes.includes(state.values.type)) {
+    return `Column type \`${state.values.type}\` does not support \`Unique: true\``;
+  }
+  if (isUnique && parseBoolean(state.values.notNull)) {
+    return 'Column cannot be both `Unique: true` and `Not null: true`';
+  }
+  if (isUnique && state.values.defaultValue) {
+    return 'Column cannot be both `Unique: true` and have a `Default value`';
+  }
+  return true;
+}
+
+function validateNotNull(notNullValue: string | undefined, state: ColumnEditState) {
+  const isNotNull = parseBoolean(notNullValue);
+  if (isNotNull && state.values.type && notNullUnsupportedTypes.includes(state.values.type)) {
+    return `Column type \`${state.values.type}\` does not support \`Not null: true\``;
+  }
+
+  return true;
+}
```

Comment:
It’s a lot of logic, hard to say if this is working as expected without tests 🙄 Should we add some? 😁